book/what-is-wallaroo.md

```diff
@@ -49,9 +49,9 @@ How does one go about building a Wallaroo application? Via our developer framewo
 - Source
 - Sink
 
-The most important of these is a `Computation`. Computations come in two varieties: stateless and stateful. A stateless computation takes some data as an input and creates some new data as an output. For example, a “double computation” might take in an integer such as 2 and output 4. A stateful computation is similar to a stateless computation except it takes an additional input: the state it will operate on. An example of a stateful computation would be a counter that keeps track of the running total of all numbers it has processed. 
+The most important of these is a `Computation`. Computations come in two varieties: stateless and stateful. A stateless computation takes some data as an input and creates some new data as an output. For example, a “double computation” might take in an integer such as 2 and output 4. A stateful computation is similar to a stateless computation except it takes an additional input: the state it will operate on. An example of a stateful computation would be a counter that keeps track of the running total of all numbers it has processed.
```

Comment:
What changed here? I see some of the other lines had trailing spaces removed but that doesn't seem to be the case here.